build.json

```diff
@@ -46,7 +46,7 @@
     {
       "Name": "Microsoft.SqlTools.SqlCore",
       "Frameworks": [
-        "netstandard2.0"
+        "net6.0"
```

Comment:
Are you updating the sqldb workload to net6 then? 

@aasimkhan30 FYI since you were asking about this as well

Reply:
Unfortunately, there are blockers on the provisioning code that is preventing the migration to netcore

Reply:
I was just about to tag you about this; net6 can be supported over there since the DacFx workload is built on it, yeah?  Seems like the path we'd want anyway since the overall .NET advice is that anything that can be .NET rather than Standard or Framework should be.

Reply:
I see. So, you plan on writing APIs in the dacfx workload for this feature, right? If that's the case using netcore is fine. @Charles-Gagnon FYI.

Reply:
Resolved offline